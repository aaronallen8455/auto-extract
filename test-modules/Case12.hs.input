{-# LANGUAGE PatternSynonyms #-}
module Case12 where

pattern Test :: ()
pattern Test = ()

test x = EXTRACT@simple (x + 3)
