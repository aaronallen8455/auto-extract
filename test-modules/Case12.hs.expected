{-# LANGUAGE PatternSynonyms #-}
module Case12 where

pattern Test :: ()
pattern Test = ()

test x = simple x

simple :: Num a => a -> a
simple x = x + 3
